DataSetExplorer/Program.cs

```diff
@@ -27,8 +34,67 @@ class Program
         static void Main(string[] args)
         {
             //MakeExcelFromProjectUseCase();
-            FindInstancesRequiringAdditionalAnnotationUseCase();
+            //FindInstancesRequiringAdditionalAnnotationUseCase();
             //FindInstancesWithAllDisagreeingAnnotationsUseCase();
+            ExportAnnotatedDataSet();
+        }
+        
+        private static void ExportAnnotatedDataSet()
+        {
+            ListDictionary projects = new ListDictionary(); // local repository path, annotations folder path
+            projects.Add("D:/ccadet/annotations/repos/BurningKnight", "D:/ccadet/annotations/annotated/BurningKnight");
+            projects.Add("D:/ccadet/annotations/repos/Core2D", "D:/ccadet/annotations/annotated/Core2d");
+            projects.Add("D:/ccadet/annotations/repos/jellyfin", "D:/ccadet/annotations/annotated/Jellyfin");
+            projects.Add("D:/ccadet/annotations/repos/OpenRA", "D:/ccadet/annotations/annotated/OpenRA");
+            projects.Add("D:/ccadet/annotations/repos/ShareX", "D:/ccadet/annotations/annotated/ShareX");
+            projects.Add("D:/ccadet/annotations/repos/ShopifySharp", "D:/ccadet/annotations/annotated/ShopifySharp");
+
+            var dataForExport = PrepareDataForExport(projects);
+            var groupedBySmells = dataForExport.GroupBy(t => t.Item1.Annotations.ToList()[0].InstanceSmell.Value);
+            
+            var enumerator = groupedBySmells.GetEnumerator();
+            while (enumerator.MoveNext())
+            {
+                var exporter = new DataSetWithMetricsExporter("D:/ccadet/annotations/annotated/Output/");
```

Comment:
Regarding your comment: Program.cs in this case is the client code of DataSetWithMetricsExporter. You are forcing the Program.cs to instantiate DataSetWithMetricsExporter twice, or rather once for each Export operation.

DataSetWithMetricsExporter would be more useful if it could be instantiated above the loop and then Export could be called multiple times.